src/docfx/build/xref/XrefResolver.cs

```diff
@@ -270,6 +270,8 @@ private static string RemoveSharingHost(string url, string hostName)
         private (Error?, IXrefSpec?, string? href) Resolve(
             SourceInfo<string> uid, FilePath referencingFile, FilePath inclusionRoot, MonikerList? monikers = null)
         {
+            var a = uid.Value;
```

Comment:
No reference of `a` defined in `var a = uid.Value;`

Reply:
Fix it in another pr.